src/nvim/log.h

```diff
@@ -67,6 +67,30 @@
        __VA_ARGS__)
 #  endif
 
+# if __linux__
+#  include <execinfo.h>
+#  define LOG_CALLSTACK(prefix) \
```

Comment:
Actually I would suggest to make a function `log_callstack` which accepts `const char *const func_name, const int line_num, const char *const prefix`. Not even a `static inline`, just ignore last entry in the trace. And make macro with the same signature.